helm-system-packages-dpkg.el

```diff
@@ -308,18 +281,26 @@ If REVERSE is non-nil, show reverse dependencies instead."
                      (mapconcat 'identity (helm-marked-candidates) " "))))
         (helm-system-packages-dpkg)))))
 
+(defun helm-system-packages-make-apt-get-command ( &rest args)
+  (let ((comm (append '("apt-get" "-q") args)))
+    (if helm-system-packages-dpkg-confirm-p
+        (append '("DEBIAN_FRONTEND=readline") comm)
+      (append '("DEBIAN_FRONTEND=noninteractive") comm))))
+
 (defcustom helm-system-packages-dpkg-actions
   '(("Show package(s)" .
      (lambda (_)
        (helm-system-packages-print "apt-cache" "show")))
     ("Install (`C-u' to reinstall)" .
      (lambda (_)
-       (helm-system-packages-dpkg-run-as-root
-        "apt-get" "install" (when helm-current-prefix-arg "--reinstall"))))
+       (apply 'helm-system-packages-run-as-root
+              (helm-system-packages-make-apt-get-command "install"
+                                                         (when helm-current-prefix-arg "--reinstall")))))
```

Comment:
Why not keeping the command as it was and insert

```
  (if helm-system-packages-dpkg-confirm-p
	    "DEBIAN_FRONTEND=readline"
    "DEBIAN_FRONTEND=noninteractive")
```

before "apt-get"?

Reply:
I don't like idea to duplicate this piece of code three times and maybe more if some new actions will be added later. May be we can get rid of apply somehow to make this simpler.

Reply:
Fair enough, then go ahead and I'll merge your changes.  Thanks a lot!